templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/MenuGroups.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <b-container>
+    <div v-for="(menu, index) in filteredMenuGroups" :key="index" class="py-1">
+      <b-card>

```

Comment:
I think I mentioned this before, by I don't see why you need a div wrapper here - what's wrong with just having the card by itself without a wrapper? Why add a div?